adapter/redisquota/driver.go

```diff
@@ -37,7 +39,9 @@ type response struct {
 
 // Get is to get a connection from the connection pool.
 func (cp *connPool) get() (*connection, error) {
+	cp.Lock()
```

Comment:
Yes, at first I also think these functions are already thread-safe. But I want to test within Mixer by myself, could you help to clarify the questions I posed before (in the PR description)? Thank you!